packages/bsky/src/hydration/actor.ts

```diff
@@ -116,6 +122,7 @@ export class ActorHydrator {
         takedownRef: safeTakedownRef(actor),
         isLabeler: actor.labeler ?? false,
         allowIncomingChatsFrom: actor.allowIncomingChatsFrom || undefined,
+        upstreamStatus: actor.upstreamStatus ?? undefined,
```

Comment:
I think thanks to grpc that `actor.upstreamStatus` could be `''` when missing, does that seem right?  If that's the case we might want:
```suggestion
        upstreamStatus: actor.upstreamStatus || undefined,
```